### DISCUSSION TITLE
Welcome to graves ontology Discussions!

### DISCUSSION THREAD

**Discussion by rwarren2** [muninn/graves#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Stephen-Gates** [comment]
👋 Hi, I'm @Stephen-Gates. I live in Brisbane, Australia 🐨🇦🇺

I recently joined [Friends of Toowong Cemetery](http://www.fotc.org.au/) and am [rebuilding](https://1871fotc.github.io/fotc/) their website. I am planning to publish [Toowong Cemetery data](https://1871fotc.github.io/fotc/about/data/) using the [Graves](https://rdf.muninn-project.org/ontologies/graves-en.html) and [Bio](https://vocab.org/bio/) ontologies. I'm also encouraging our Local Government to publish spatial data about the cemetery so I can make maps for self-guided walks around the cemetery, and use polygons instead of points in the Graves data. 

I am new to Linked Open Data and am busy learning more about it so I contribute to this project. I'm choosing a data publishing platform and am leaning towards [Ontotext GraphDB](https://www.ontotext.com/products/graphdb/) but suggestions are welcome. Right now, I'm curious about RDF-star (_but I am probably getting ahead of myself_).

Big thanks to @rwarren2 and other contributors for the work done so far ✨

Looking forward to having conversations here. 💬
